src/api/index.js

```diff
@@ -0,0 +1,22 @@
+const API_END_POINT = "https://kdt-frontend.programmers.co.kr";
+const X_USERNAME = "sukvvon";
+
+export const request = async (url, option = {}) => {
+  try {
+    const res = await fetch(`${API_END_POINT}${url}`, {
+      ...option,
+      headers: {
+        "Content-Type": "application/json",
+        "x-username": X_USERNAME,
+      },
+    });
```

Comment:
P1;

이렇게 하게 된다면 headers를 변경하고 싶어도 변경하지 못할 것 같아요.
아래와 같이 headers 또한 수정 가능하게 해두면 어떨까요?

```suggestion
    const res = await fetch(`${API_END_POINT}${url}`, {
      ...option,
      headers: {
        "Content-Type": "application/json",
        "x-username": X_USERNAME,
        ...option?.headers,
      },
    });
```